packages/config/config/devices/0x011a/zwn-bpc.json

```diff
@@ -1,7 +1,7 @@
 {
 	"manufacturer": "Wenzhou MTLC Electric Appliances Co., Ltd.",
 	"manufacturerId": "0x011a",
-	"label": "ZWN-BPC-PLUS",
+	"label": "ZWN-BPC",
```

Comment:
I think this and the next deleted file isn't correct. AFAIK there is a ZWN-BPC and a ZWN-BPC-PLUS.

Reply:
That's why we have an open Issue to make labels conditional. There is no difference here except the device label. I've stetted this back for now.